GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/ContentTypes/govuksectionbreak.config

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<ContentType Key="5d33cd38-b819-4afd-9d4a-499ee73799ab" Alias="govukSectionBreak" Level="2">
+  <Info>
+    <Name>Section break</Name>
+    <Icon>icon-inactive-line color-black</Icon>
+    <Thumbnail>folder.png</Thumbnail>
+    <Description>A thin grey horizontal line used to break up sections of text</Description>
```

Comment:
I'd like the description for this component to explain the semantic meaning of it for accessibility (I've linked to more about that from another comment). A good description would be "A thematic break between sections of content. DO NOT use a section break for purely aesthetic reasons."